src/pages/applications.js

```diff
@@ -0,0 +1,105 @@
+            <li>Are a person, not a brand or company account.</li>
+          </ul>
+      </div>
+    </section>
+    <section id="process">
+      <div>
+        <h2>The Process</h2>
+        <p>
+          <ul>
+            <li>Hit the Apply Now button at the top of this page</li>
+            <li>Fill in the GitHub issue to the best of your ability</li>
+            <li>The Live Coders get alerted of your application</li>
+            <li>Any team member can give their opinions on your stream and community.</li>
+            <li>Taking all member opinions, the minimum criteria and looking closely at your VoDs 
+              or live streams, we review your application.</li>
+            <li>A decision is made and confirmed by at least one other Live Coder</li>
+            <li>You the applicant are told if you are successful at becoming a Full team member, 
+              an Intern member in the nurture program or rejected.</li>
+            <li>REJECTION: There have only ever been 3 Rejections in the past if the broadcaster 
+              meets the minimum criteria.</li>
+            <li>FULL TEAM MEMBERSHIP: Invited tot he Twitch Team, Full Access to the discord server, 
+              access to all the perks of being in the community</li>
+            <li>INTERN MEMBERSHIP: You didn't quite make the minimum criteria, however we saw 
+              something we could nurture into moving into a Full Team position.</li>
+          </ul>
+        </p>
+      </div>
+    </section>
+    <section id="why">
+      <div>
+        <h2>Why this page exists</h2>
+        <p>
+          After a year and a half The Live Coders has grown from the initial intake to 190+ members. That's 
+          almost 10 new members per month. With that, we want to be more open about the application process
+          and make what we do more transparent to all. This makes sure you get correct and timely feedback 
+          on your application. 
+          We are improving the process continuously and this page is a step in that direction.  
+        </p>
+      </div>
+    </section>
+    <section id="faq">
+      <div>
+        <h2>Frequent Questions</h2>
+        <h3>I haven't heard from you and my application is taking a long time. WHY?</h3>
+        <p>We are aiming to look at applications on a 2 week cycle at this time. It was previously 
+          monthly and we missed some. If we don't get back to you please feel free to comment on your GitHub issue.</p>
+        <h3>I meet the minimum criteria, why wasn't I accepted?</h3>
+        <p>Some of the criteria is looked is opinion based, like "polite and inviting" for example. 
+          This is why we need at least two opinions on each application. If you don't quite hit the criteria,
+           we are now inviting you to a nurture program, which give you access to team members and allows you to grow.</p>
+        <h3>What is the Nurture Program</h3>
```

Comment:
Since the term/level of "Intern Membership" is used above, re-using the term might be helpful for the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> folks looking for information based on the term "mentor"

Reply:
mentioned Nurturing more and coming in as an intern